tests/cleanup/remove_old_objects_openstack.sh

```diff
@@ -24,6 +24,9 @@ rlJournalStart
             rlLogInfo "OS_PASSWORD is configured"
         fi
 
+        export OS_PROJECT_NAME="${OS_PROJECT_NAME:-$OS_USERNAME}"
+        rlLogInfo "OS_PROJECT_NAME=$OS_PROJECT_NAME"
```

Comment:
Since you say it is recommended in the docs that means OS_TENANT_NAME should be changed to OS_PROJECT_NAME everywhere else :-(

Reply:
OS_TENANT_NAME replaced with OS_PROJECT_NAME in tests/cli/test_build_and_deploy_openstack.sh too.
OS_TENANT_NAME is not used anywhere else.